dev/dependencyList

```diff
@@ -30,6 +30,7 @@ arrow-vector/16.0.0//arrow-vector-16.0.0.jar
 checker-qual/3.42.0//checker-qual-3.42.0.jar
 classgraph/4.8.138//classgraph-4.8.138.jar
 commons-codec/1.15//commons-codec-1.15.jar
+commons-io/2.11.0//commons-io-2.11.0.jar
```

Comment:
LICENSE-binary and NOTICE-binary should be updated too

Reply:
Addressed.